src/WorkOS.net/Services/SSO/SSOService.cs

```diff
@@ -34,9 +34,11 @@ public SSOService(WorkOSClient client)
         /// <returns>An Authorization URL.</returns>
         public string GetAuthorizationURL(GetAuthorizationURLOptions options)
         {
-            if (options.Domain == null && options.Provider == null && options.Connection == null)
+#pragma warning disable CS0618 // GetAuthorizationURLOptions.Domain' is obsolete: 'The Domain property is deprecated. Please use Organization instead.
```

Comment:
We have warnings set as errors, so we need to disable the warning to access the deprecated `Domain` property.